build/Dockerfile-zb

```diff
@@ -1,10 +1,12 @@
 # ---
 # Stage 1: Install certs, build binary, create default config file
 # ---
-FROM ghcr.io/project-zot/golang:1.20 AS builder
+FROM --platform=$BUILDPLATFORM ghcr.io/project-zot/golang:1.20 AS builder
+
+ARG TARGETOS
+ARG TARGETARCH
 ARG COMMIT
-ARG OS
-ARG ARCH
+
 RUN mkdir -p /go/src/github.com/project-zot/zot
 WORKDIR /go/src/github.com/project-zot/zot
 COPY . .
```

Comment:
The make command below needs an update the replace the old OS and ARCH variables with the new ones.